src/logging.cc

```diff
@@ -459,6 +476,7 @@ class LogFileObject : public base::Logger {
   // can avoid grabbing a lock.  Usually Flush() calls it after
   // acquiring lock_.
   void FlushUnlocked();
+  void CheckHistoryFileNum();
```

Comment:
A brief comment describing the purpose of the method would be helpful here.

Reply:
done